api/v1alpha1/etcdcluster_types.go

```diff
@@ -36,6 +36,24 @@ type EtcdClusterSpec struct {
 	Version string `json:"version"`
 	// StorageSpec is the name of the StorageSpec to use for the etcd cluster. If not provided, then each POD just uses the temporary storage inside the container.
 	StorageSpec *StorageSpec `json:"storageSpec,omitempty"`
+	// Tls is the TLS certificate configuration to use for the etcd cluster and etcd operator.
+	Tls *TLSCertificate `json:"tls,omitempty"`
```

Comment:
Suggestion to use Go-style casing for the variable name:

```suggestion
	// TLS is the TLS certificate configuration to use for the etcd cluster and etcd operator.
	TLS *TLSCertificate `json:"tls,omitempty"`
```

Reply:
I am aware of the related naming checking linter used by etcd.

Actually I intentionally used "Tls" instead of "TLS", the nuance is that I will definitely use `TLSXXX`, but there isn't the suffix `XXX` here.

Anyway, updated per your suggestion for now.

Reply:
Got it. We can rename then later. I wonder why the linter didn't catch it up.